web/src/lib/components/shared-components/leaflet/asset-marker-cluster.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts" context="module">
+	import { createContext } from '$lib/utils/context';
+	import L from 'leaflet';
+
+	const { get: getContext, set: setClusterContext } = createContext<() => L.MarkerClusterGroup>();
+
+	export const getClusterContext = () => {
+		return getContext()();
+	};
+</script>
+
+<script lang="ts">
+	import 'leaflet.markercluster';
```

Comment:
this is a very interesting way to import a functionality of a library. Is this a recommended way from the library? can we use something like `import {markerclsuter} from 'leaflet'`?

Reply:
I personally don't think this works because of the separation of TS types and actual implementation here. But I'll check tomorrow.

Reply:
Nope, does not work :/